EssentialFeed/EssentialFeediOS/Image Comments UI/Controllers/ImageCommentsViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ImageCommentsViewController.swift
+//  EssentialFeediOS
+//
+//  Created by Lukas Bahrle Santana on 02/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public protocol ImageCommentsControllerDelegate {
+	func didRequestImageCommentsRefresh()
+}
+
+public final class ImageCommentsViewController: UITableViewController, ImageCommentsView, ImageCommentsLoadingView, ImageCommentsErrorView {
+	
+	private(set) public var errorView: ErrorView = ErrorView(frame: CGRect(x: 0, y: 0, width: 320, height: 40))
```

Comment:
The frame shouldn't be necessary here. Remove it.

The property can also be lazy.

```suggestion
	private(set) public lazy var errorView = ErrorView()
```